### ISSUE TITLE
Dark Mode - Dark Reader Visibility Issue

### ISSUE THREAD

**Issue by poisonouschimp** [codemirror/CodeMirror#6138]
Using Chrome on Dark Mode and Dark Reader addon highlighted text is very difficult to see. I am not able to overwrite the css styling via Dark Reader for CodeMirror. 

I use the following CSS and it works on all other sites other than CodeMirror sites. 

*
CSS
::selection {
color: black;
background: yellow;
}

<img width="509" alt="highlighting" src="https://user-images.githubusercontent.com/8282735/73979488-1d071180-48e3-11ea-81f7-6bd62b4d1caa.PNG">

**Comment by bruchmann** [comment]
This is because CodeMirror doesn't use the _native_ selection provided by the browser and renders its own selection. According to the [docs](https://codemirror.net/doc/manual.html#styling) your selector should include `.CodeMirror-selected`.

**Comment by poisonouschimp** [comment]
@bruchmann  I changed it to 
```
.CodeMirror-selected {
color: black;
background: yellow;
}

::selection {
color: black;
background: yellow;
}
```
When I use the search for a word like CTRL + F it does highlight it correctly now. When I highlight it with my cursor though it does something weird. If I just highlighted it inside the window it is still transparent. If I then click outside the window to the main CMS it will then change my transparent selection to my style. Is there anyway to get it so I don't have to click outside?
